packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -247,6 +248,38 @@ describe("Generator utils", () => {
       assert.equal(zip.getEntries().length, 1);
     });
   });
+
+  it("download directory get file info error", async () => {
+    const axiosStub = sandbox.stub(axios, "get");
+    axiosStub.onFirstCall().resolves({ status: 403 });
+    try {
+      await downloadDirectory(
+        "https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/bot-sso",
+        tmpDir
+      );
+    } catch (e) {
+      assert.exists(e);
```

Comment:
Consider asserting the error status and verifying that the error being stubbed is the one being tested for.

Reply:
Thanks. Added.